src/javascript-node-postgres/devcontainer-template.json

```diff
@@ -1,6 +1,6 @@
 {
     "id": "javascript-node-postgres",
-    "version": "3.1.0",
+    "version": "3.1.1",
```

Comment:
Can we make same changes (as commented before) for this Template as well?

Reply:
done